environment_setup.sh

```diff
@@ -88,38 +88,30 @@ if confirmupdate "Would you like to install local development programs like PHPS
   echo "Installing alternate PHP versions (5.6, 7.1)"
   echo $'\n'
 
-  brew unlink php70
-
-  brew install php56
-  brew install php56-opcache
-  brew install php56-mcrypt
-  brew install php56-xdebug
-  brew install php56-redis
-
-  brew unlink php56
-
-  brew install php71
-  brew install php71-opcache
-  brew install php71-mcrypt
-  brew install php71-xdebug
-  brew install php71-redis
-
-  brew unlink php71
-  brew link php70
+  brew install php@56
+  brew install php@70
+  brew install php@71
 
   echo $'\n'
   echo "Configuring PHP"
   echo $'\n'
 
-  for VER in 5.6 7.0 7.1
+  echo $'\n'
+  echo "Installing xdebug, redis, and mcrypt with pecl"
+  echo $'\n'
+  pecl install xdebug
+  pecl install redis
+  pecl install mcrypt
+
+  for VER in 5.6 7.1 7.2
   do
     $(brew --prefix gettext)/bin/envsubst < config/php-ts.ini > $(brew --prefix)/etc/php/$VER/conf.d/php-ts.ini
   done
 
   echo $'\n'
   echo "Starting PHP7 FPM process."
   echo $'\n'
-  brew services start php70
+  brew services start php72
```

Comment:
This is just `php`, not `php72` - and there are other brew services for the old versions like `php@5.6`